### PR TITLE
EN-12804: documented apr calculation

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
#### Description of the pull request (what is new / what has changed)
documentation about staking providers - APR computation

#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [ ] update documentation that is not relevant anymore
- [x] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

**Review by AdoAdoAdo — CHANGES_REQUESTED** [review]

File: docs/validators/apr.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
+---
+id: staking-providers-apr
+title: Staking Providers APR
+---
+
+## Introduction
+
+By using the [Delegation Manager](/validators/delegation-manager/) system smart contract, a new staking provider can be 
+set up. According to the initial deposits (half the price of a node) + delegations from other users (or even the owner itself)
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/validators/apr.md:9]: fixed

File: docs/validators/apr.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+the staking contract can spawn new nodes. Currently, a node costs 2500 EGLD, so, for example, if a staking contract
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/validators/apr.md:10]: fixed

File: docs/validators/apr.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+gathered 7500 EGLD it can spawn 3 new nodes.
+
+### Base stake and top-up
+
+As stated, a validator node requires 2500 EGLD. So multiple nodes would mean a number of tokens that is divisible by 2500. 
+The upper difference is considered top-up. Also, the staking provider owner can choose to keep the tokens as top-up, even
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/validators/apr.md:16]: fixed

File: docs/validators/apr.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+## Introduction
+
+By using the [Delegation Manager](/validators/delegation-manager/) system smart contract, a new staking provider can be 
+set up. According to the initial deposits (half the price of a node) + delegations from other users (or even the owner itself)
+the staking contract can spawn new nodes. Currently, a node costs 2500 EGLD, so, for example, if a staking contract
+gathered 7500 EGLD it can spawn 3 new nodes.
+
+### Base stake and top-up
+
+As stated, a validator node requires 2500 EGLD. So multiple nodes would mean a number of tokens that is divisible by 2500.
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/validators/apr.md:15]: fixed

File: docs/validators/apr.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+This means that for each year the estimated rewards for a validator will change. This has to be taken into account 
+when computing the APR.
+
+The configuration for the inflation rate can be found [here](https://github.com/ElrondNetwork/elrond-config-mainnet/blob/master/economics.toml) (`YearSettings`).
+
+The approximated inflation rate is as follows:
+
+| Year | Inflation rate |
+|------|----------------|
+ | 1    | 10.84%         |
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/validators/apr.md:52]: fixed

File: docs/validators/apr.md [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+In accordance to the Mainnet's [configuration](https://github.com/ElrondNetwork/elrond-config-mainnet/blob/master/economics.toml#L35)  (`ProtocolSustainabilityPercentage`).
+at each epoch change, when new tokens are distributed among the validators, 10% of the value goes to Protocol Sustainability Address.
+
+This also has to be taken into account when calculating the APR.
+
+
+## Rewards calculation
+
+When wanting to calculate the APR (Annual Percentage Return) of a Staking Provider, there are multiple factors that have
+to be taken into account, such as total value locked at Network-level, the inflation based on the current year, and so on.
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/validators/apr.md:76]: fixed

File: docs/validators/apr.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+
+### Network Top-Up rewards
+
+The formula for determining the rewards received by the validators for the top-up in a given epoch is:
+
+$$
+topUpRewards(e) = \frac{(2 * topUpRewardLimit(e)}{\pi} * atan(\frac{eligibleCumulatedTopUp(e)}{p})
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/validators/apr.md:83]: fixed

File: docs/validators/apr.md [hunk 8]
```diff
@@ -0,0 +1,225 @@
...
+$$
+
+Where:
+- `e` represents the given epoch
+- `topUpRewardLimit(e)` represents the maximum top-up rewards that can be distributed in the given epoch
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/validators/apr.md:88]: the more details, the better. added

File: docs/validators/apr.md [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+where the given top-up rewards reach ½ of the top-up rewards set limit. It is currently set to 2M EGLD.
+
+### Network base rewards
+
+$$
+baseRewards(e) = blocksRewards(e) - topUpReward(e)
+$$
+
+Where:
+- `blocksRewards(e)` represents the rewards received by the validators by either signing or proposing blocks during the epoch that are now
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/validators/apr.md:100]: fixed

File: docs/validators/apr.md [hunk 10]
```diff
@@ -0,0 +1,225 @@
...
+part of the canonical chain
+- `topUpReward(e)` is computed above
+
+
+## APR calculation
+
+After determining the base and the top-up rewards for an epoch, the APR can be calculated for a Staking Provider.
+
+First, we have to determine the maximum rewards that can be reached in ideal situations (no roll-back in an epoch).
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/validators/apr.md]: fixed

**Pull request merged**